### ISSUE TITLE
Login view does not display properly in a non-widescreen window 

### ISSUE THREAD

**Issue by AndruC** [HumanityInc/Humanity#6]
![not-flexible](https://cloud.githubusercontent.com/assets/843112/6499222/933bc9da-c2c5-11e4-9f47-2fcd7d32e8e1.PNG)

**Comment by HumanBe1ng** [comment]
I'm sure Alex has already seen this but I'll make sure it gets fixed. We will have an online tool soon where you can submit this and where everyone can view the newest suggestions from all users and the status reports on each - this will be built into our network - you can vote on top most wanted new features, and all sorts of things. Yes, I can see how that can be a problem with the current layout we have. We have yet to begin testing anything we've made on a wide range of systems - for now it's just the usual suspects (Win / Mac / Linux with Chrome / Firefox / Safari, that sorta thing, plus a few mobile operating systems). In the very near future we will run everything we have on every device imaginable and debug until it's PERFECT. Also, there is a similar problem to do this if you view our site vertically on mobile - right now it only works horizontally. It seems that for vertical view, we will have to completely re-adapt our landing page for that. It will be done, not to worry. We are perfectionists - we never stop. :)

**Comment by AndruC** [comment]
Max, I'm just making note of the issues I find. This is my contribution. You do not have to defend each issue with promises of the future.

You're micromanaging. Let profess0r-x handle the bug reports.

**Comment by rjamesnw** [comment]
I suggest you guys look into responsive design using CSS media queries. Twitter's bootstrap project is a good example.

**Comment by stephen-cornwell** [comment]
I'd second the call to use bootstrap. Mobile should never be a second thought with web applications like this.

**Comment by rjamesnw** [comment]
I just finished a mobile project for a billion dollar global company (they wanted me to help pioneer it), and I used bootstrap with HTML5/CSS (using an "app.manifest file for caching as well, and to support native mode)  to create a web app that has a full menu, etc., in desktop mode, and looks great in compact mode for _any_ mobile device (since it "grows" and "shrinks" as needed).  HTML5/CSS with media queries is the way I believe most HTML5 apps of the future should be IMHO.  I won an award for this project as well, btw. ;)

**Comment by rjamesnw** [comment]
Side note:  You can also wrap the app in CocoonJS or PhoneGap and create a native app from your pages as well.  I'm doing it now using CocoonJS for the app I menioned above.  Runs/looks great too.

**Comment by mashahuman** [comment]
<3

**Comment by Jarema** [comment]
Twitter bootstrap, Foundation, Materialize - there are plenty of responsive webdesign frameworks. Using one is just a good practice - there is no need to reinvent the wheel. rjamesnw is definately right.

**Comment by lamarrotems** [comment]
Any updates? 

Getting a feeling that this project isn't really going to happen.

**Comment by lamarrotems** [comment]
I thought it was just the final few tweaks before launch :{

**Comment by lamarrotems** [comment]
Any updates from Alex? I was thinking it would be up by end of March for sure!

**Comment by stephen-cornwell** [comment]
Is Alex the developer? As far as I'm aware no one has shown their face here at all aside from Max. And judging by the single inconsequential commit in the last two months (a couple of hours work at best) and the current state of the code I would guess they're a long long way off from launching.

**Comment by lamarrotems** [comment]
@StephenCornwell Well, Max has talked about a developer, but no one has ever spoken with him. I guess that is VERY odd for a developer to be completely absent from the projects development page - especially with Humanity's emphasis on "open source". 

This image from July 2014 was part of a post saying the network would be up in a week: 

http://imgur.com/6JF1A6a

So we are almost at 1 year and haven't seen any progress. At this point no one believes anything Max says - only when their is a functioning public website that anyone can access  will people believe him. 

Otherwise someone like myself could just claim I have a 10 billion dollar company with a social network to be launched next month with amazing features and 100,000 hours of coding completed. Without a functioning website that anyone can access the words are meaningless. ESPECIALLY coming from a billion dollar company.

One month ago he said there were only tiny cosmetic changes necessary - and those would and could be fixed in a few hours. Of course he has either locked those threads or edited the comments. Taking screenshots of this comment as proof.

**Comment by HumanBe1ng** [comment]
Guys I'm sorry I don't have time for this - I just spent the past ONE AND A HALF OURS BEGGING Steve Wozniak to join us - THESE are the sorts of things I spend my time on - I do NOT have time to answer bullshit questions about "why there was one commit in 2 months" when the graph shows TEN, or "why were you the only person to show your face," when THREE people showed up. Maybe you guys are bored and have nothing to do, but _I_ DO. Or "Is Alex the developer" when you can see a commit from him just DAYS ago. Wow. Finally, to answer the one ACTUAL question, none of the delays are Alex's fault - they are 100% mine. Right now he has been waiting on some directions on fixing minor crap (the stuff I mentioned earlier, small cosmetic stuff only) for TWO WEEKS. He has sent this to me AGAIN and AGAIN. But I am simply TOO BUSY. You guys have to understand that we expect LESS THAN ONE PERCENT of our future revenue to come from Touch and Empower - Touch has NO ADS, and Empower we will eventually make FREE. I am focusing on THE OTHER 99 - I am trying to build a TRILLION dollar company here (as insane as this might sound, we are aiming at a trillion dollar valuation within 20 years, and are expecting to become around the 5th to 7th trillion dollar company by then). Given the fact that Apple went from almost NOTHING (few billion valuation, little more than we're worth today) to 700 BILLION in TEN YEARS, we think this is 100% possible. I am working on THAT - my focus is NOT a tiny beta, on web only, of 2 of TWENTY online services - when you take ALL of our services into account on ALL the platforms, this is not even ONE PERCENT of our code - it is probably closer to ONE TENTH OF ONE PERCENT. And then there is the HARDWARE. I suggest you guys keep an eye on HumanityTechnologies.com - THAT is where the action is, THAT is where you can see a BROADER PICTURE of what we are doing. By the way, we are now in the process of hiring a 3D artist in Ukraine, we are opening an office in Lviv that will initially consist of our frontend (AngularJS) developer, one or two mobile app devs, our sysadmin, and one or two secretaries. There will also be an editor in charge of summarizing tech and business news for easy consumption by our execs, and of course this 3D artist. And another person I am forgetting.. This is our SECOND location, after Toronto. The Silicon Valley location is also opening VERY shortly - I am about to move there from Canada, it will be our new home. Our secondary engineering department and our ENTIRE customer support department will be in Ukraine, our primary engineering department, finance, and design will be in California. Toronto will only have the regional reps same as every other country (to deal with local service issues for Taste, Live, Ride, and so on). Anyway guys I gotta go, I've got no time for this. I will send Alex everything he needs tomorrow, and I think he now has something like half a day of work left to implement it.. he is basically DONE. Then we run this thing for 2-3 weeks, get as much feedback as possible, put MORE emphasis on what the users LOVE the MOST, correct anything they might dislike (can't imagine what, but IF), then we take that usage data and apply it to our MOBILE APPS, which now that the backend is done, are basically a no-brainer: it only takes a couple of DAYS to make a beta Android app for instance, with just the Discover grid, content sharing and messaging - the rest we will add later. Same for iOS and Glass. I won't give you any deadlines because I have zero respect for them, but I will tell you that I personally expect to launch the beta THIS WEEK (well, it was launched a LONG time ago, but it was more like an alpha - this will be beta 1) and the apps by END OF THIS COMING MONTH. Oh, the reason I mentioned the 3D artist.. We have decided to release the first global exclusive renderings with a major publisher (probably we will give them to WIRED) in print and online, BUT, we will be doing that this SUMMER, so.. we thought that to WET YOUR APPETITE, we should release the renderings (and movies) of the PHONE first, and then everybody can wait to see the tablets and most importantly HEADSETS this summer. I promise this will BLOW YOUR MIND GUYS - this is the BEST looking phone, the BEST looking tablet, the BEST looking headset in the world today. BAR NONE. I am locking this thread because I no longer have time to waste on speaking with you people: this open source project has brought ZERO benefit to our company - we are PAYING out of our pocket to write and release the code, spending time to push it here, and then ON TOP OF THAT we have to waste our time doing THIS.. To WHAT end? WHERE is the benefit for us? I see none. If things continue this way we will end our cooperation (WHAT cooperation?!) with the open source community, and just release our code every month and then you guys can do, or not do with it whatever you want.

P.S. In response to the post above that just showed up as I was typing this - no, not "nobody" has spoken to Alex. More bullshit. You DO realize this person has an EMAIL, Skype, AND Facebook? So maybe YOU haven't spoken to him, but there is an ENTIRE WORLD out there that is NOT you. And even more bullshit towards the end - "1 year without any progress" - SCREW YOU.. Sorry, I've had enough of these lies, I will not justify that FOOLISHNESS with an actual response. Please go back to trolling on reddit, or whatever it is you do all day. :)

P.P.S. I am locking this issue because I will NOT waste any further time on this. If you have questions, READ THE NEWS at HumanityTechnologies.com, and if that STILL doesn't answer it, POST YOUR QUESTION THERE. The issue this thread was ACTUALLY started for (the small screen thing) has been added to our log and will be dealt with in due time (not soon - it is not a priority). Have a nice day everybody! :)
